### PR TITLE
Fix a bug and add Invincibility

### PR DESCRIPTION
**Pull request by pointillion**
Invincibility doesn't work, though.

### CODE REVIEW

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                }
+                targetAllyAoe().act {
+                    applyBuff(CounterHealBuff, values2, times2)
+                }
+            }
+        },
+        ActType.ClimaxAct.blueprint("End of the Journey") {
+            Act {
+                targetAoe().act {
+                    attack(values1)
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt:46]: Hit count is incorrect. As a general style rule, all acts should list a hit count for attacks even if the hit count is 1.

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    applyBuff(InvincibilityBuff, turns = times2)
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt:47]: This is buffing the enemy.

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/PlayerDress.kt [hunk 3]
```diff
@@ -54,6 +54,7 @@ val playerDresses = listOf(
     JudgementShizuha,
     JusticeNana,
     KondoIsamiIchie,
+    LittlePrinceMisora,
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/PlayerDress.kt:57]: Update the dress.yaml file in the server module when adding a new dress.

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/stage/strategy/interactive/InteractiveStrategy.kt [hunk 4]
```diff
@@ -133,9 +133,13 @@ class InteractiveSimulationController(val maxTurns: Int, val seed: Int, val load
         }
     }
 
+    // Safari doesn't support lookbehinds for the obvious escape processing,
+    // so use a lookahead instead
     private fun String.splitArgs() = trim()
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/stage/strategy/interactive/InteractiveStrategy.kt:138]: Argument parsing should really be reworked altogether at some point, but this is fine for now.

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                    attack(values1)
+                }
+                targetAllyAoe().act {
+                    applyBuff(CounterHealBuff, values2, times2)
+                }
+            }
+        },
+        ActType.ClimaxAct.blueprint("End of the Journey") {
+            Act {
+                targetAoe().act {
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt:45]: Misora is front 3 targeting.

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+                }
+            }
+        },
+        ActType.Act2.blueprint("Snake Bite") {
+            Act {
+                targetFront().act {
+                    attack(values1)
+                }
+                targetAllyAoe().act {
+                    applyBuff(DexterityUpBuff, values2, times2,)
```
Comment (qwewqa) [inline relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt:29]: Style should match that of other dress presets. Arguments should be put on separate lines, and named arguments should be used.

**Review by qwewqa — CHANGES_REQUESTED** [review]

File: relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt (hunk 2 above)
Comment (pointillion) [reply relive-simulator-core/src/commonMain/kotlin/xyz/qwewqa/relive/simulator/core/presets/dress/back/space/Misora.kt:47]: Oops.

**Review by qwewqa — APPROVED** [review]

**Pull request merged**